src/server/server_family.cc

```diff
@@ -90,6 +89,17 @@ ABSL_FLAG(ReplicaOfFlag, replicaof, ReplicaOfFlag{},
           "to replicate. "
           "Format should be <IPv4>:<PORT> or host:<PORT> or [<IPv6>]:<PORT>");
 
+ABSL_FLAG(string, s3_endpoint, "", "endpoint for s3 snapshots, default uses aws regional endpoint");
+// Disable EC2 metadata by default, or if a users credentials are invalid the
+// AWS client will spent 30s trying to connect to inaccessable EC2 endpoints
+// to load the credentials.
+ABSL_FLAG(bool, s3_ec2_metadata, false,
+          "whether to load credentials and configuration from EC2 metadata");
+// Disables S3 payload signing (even over HTTP). This reduces the latency and
+// resource usage when writing snapshots to S3, at the expense of security.
+ABSL_FLAG(bool, s3_disable_payload_signing, false,
```

Comment:
the name a bit confusing. maybe reverse the name to "s3_payload_signing" and flip the default to "true"?

Reply:
yep updated